Hyperledger Identus Q3 2024 report (#309)
hyperledger/toc: project-reports/2024/2024-Q4-Hyperledger-Identus.md

---
layout: default

> title: 2024 Q3 Hyperledger Identus

Reviewer: ```suggestion
title: 2024 Q4 Hyperledger Identus
```

With Q4 reports, we are moving to the new TAC governance site. Could you please close this PR and re-open a PR in the https://github.com/lf-decentralized-trust/governance report and add the report to the `tac/project-updates` folder?

Reviewer 2: I've created this PR  https://github.com/LF-Decentralized-Trust/governance/pull/44/files

9. Improved documentation with the Quick Start Guide and Tutorials

# Current Plans

As the IOG team that acted as maintainers of Identus has scaled down, the objective during this quarter will be to encourage more participation from the community in both development, governance and community engagement
The team will be focusing on simplifying the deployment and documentation of our stack, we will no longer be pursuing some of the features we were working on which were more intended for Enterprise use-cases, but instead we are going to focus on what our community members need, one very important one is having diversity of choice on the Prism NODE, universal resolver and VDR

The Community is encouraged to log feature requests, issues etc and this is then incorporated into a roadmap that is transparent for everyone to see https://github.com/orgs/hyperledger/projects/48

# Maintainer Diversity

We currently have 7 maintainers from which 3 are from outside of IOG, and we are constantly working on onboarding new members to the community by making workshops, community calls and partnering with some projects

# Contributor Diversity

> At the moment, the main contributors of source code is IOG at a 75% of the contributions.

Reviewer: ```suggestion
At the moment, the main contributors of source code are from IOG at 75% of the contributions.
```

Author: pushed, thanks!

---
layout: default
title: 2024 Q3 Hyperledger Identus
parent: 2024
grand_parent: Project Updates
nav_exclude: true
---

Created by Javier Ribó with input from other maintainers

# Project Health

The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.

> The project can now be found on the LFX dashboard which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.

Reviewer 2: can you link the [LFX Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true)

Author: Link is now added u can solve the conversation!

# Questions/Issues for the TOC

> Is there a way to access funds for open source contributions and development? 

Reviewer 2: where did this question come from?

Author: From the previous report in Identus, removed it anyway its now showing N/A
